recipes/workflows/multi-workflow-digest.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: "Multi Workflow Digest"
+description: "Send a digest notification email by batching different types of notifications from multiple workflows"
+---
+
+## Introduction
+
+This example demonstrates how to send a digest notification email by batching different types of notifications from multiple workflows using the Novu Framework. The digest notification email is sent to the user with a summary of all the notifications that have been triggered by different workflows.
+
+**Use case**: Linear is a project management tool that allows users to create and manage projects and tickets. When someone comments on the ticket, someone assigns the ticket to the user or there can be more events for which the user has turned on the Notification, the user receives a notification. The user can receive multiple notifications in a short period. To avoid spamming the user with multiple emails, Linear sends a digest notification email that contains a summary of all the notifications.
+
+Source Code on GitHub: [Multi Workflow Digest](https://github.com/novuhq/examples/tree/main/novu-workflows/react/multi-workflow-digest)
+
```

Comment:
Should we add a <Note> here, that says that this will not work in studio until the user has synced the workflows to dev env? Or is it working without it?

Reply:
it is working without syncing. I will add information also